resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -289,6 +289,12 @@ private[spark] object Config extends Logging {
       .booleanConf
       .createWithDefault(true)
 
+  val KUBERNETES_FILE_UPLOAD_PATH =
+    ConfigBuilder("spark.kubernetes.file.upload.path")
+      .doc("HCFS path to upload files to, using the client scheme:// in cluster mode.")
```

Comment:
`HCFS path where files with the client:// scheme will be uploded to in cluster mode.`